docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/abortion_miscarriage_ectopic_pregnancy_causes.rst

```diff
@@ -1,11 +1,11 @@
-.. _2021_cause_partial_term_pregnancy_causes_mncnh:
+.. _2021_cause_abortion_miscarriage_ectopic_pregnancy_causes_mncnh:
 
-================================================
-Partial term pregnancy maternal disorders causes
-================================================
+======================================================================
+Abortion, miscarriage, and ectopic pregnancy maternal disorders causes
```

Comment:
I've done the following renames everywhere:

- "term length" -> "broad pregnancy outcome"
- "partial term pregnancy" -> "abortion/miscarriage/ectopic pregnancy"
- "full term pregnancy" -> "live/stillbirth pregnancy" (except on the facility choice model page, where "full term" was used as the opposite of "preterm")